Automattic/jetpack
# Dashboard: Update jQuery usage to comply with jQMIGRATE recommendations

Cleans the console a bit by addressing some messages from jQuery migrate.



#### Changes proposed in this Pull Request:

* Updates to use .on instead of .click shorthand in AdminNotices component
* Updates to use .on instead of .bind shorthand in jetpack-jitm file
* Updates to use ​.on instead of .click shorthand in jetpack-jitm file

#### Does this pull request change what data or activity we track or use?
no
#### Testing instructions:

* Build the jitm package `jetpack build packages/jitm`
* Build the jetpack page `jetpack build plugins/jetpack`
* Visit the Jetpack dashboard
* Confirm you no longer see warnings from JQUERYMIGRATE coming from `jetpack-jitm.js` or AdminNOtices.
  ![image](https://user-images.githubusercontent.com/746152/151559143-c7264792-cef6-4575-a965-1d0990ef6783.png) 
  ![image](https://user-images.githubusercontent.com/746152/151559169-45ccf85c-4012-4db3-9f12-42319d93c431.png)
